torch/_export/serde/serialize.py

```diff
@@ -459,7 +459,7 @@ def serialize_sym_op_inputs(self, args) -> List[NamedArgument]:
         return serialized_args
 
     def serialize_inputs(
-        self, target: torch._ops.OpOverload, args, kwargs=None, include_default_values=True
```

Comment:
@SherlockNoMad I think we should just remove this argument `include_default_value` and the behaviour should always be excluding default argument values out of serialization. I don't think executorch would be an issue here and we can find ways to handle them.

Reply:
ok.